File: app/src/main/java/cc/lixiaoyu/wanandroid/ui/activity/AboutActivity.java
package cc.lixiaoyu.wanandroid.ui.activity;




import android.support.v7.app.ActionBar;
import android.support.v7.widget.Toolbar;
import android.webkit.WebView;

import butterknife.BindView;
import cc.lixiaoyu.wanandroid.R;
import cc.lixiaoyu.wanandroid.base.BaseSwipeBackActivity;

public class AboutActivity extends BaseSwipeBackActivity {
    @BindView(R.id.about_toolbar)
    Toolbar mToolbar;
    @BindView(R.id.about_webview)
    WebView mWebView;

    @Override
    protected void initData() {

    }

    @Override
    protected void initView() {
        setSupportActionBar(mToolbar);
        ActionBar actionBar = getSupportActionBar();
        actionBar.setDisplayHomeAsUpEnabled(true);
        actionBar.setHomeButtonEnabled(true);

        //初始化webview
        mWebView.loadUrl("file:///android_asset/about_us.html");
        mWebView.setHorizontalScrollBarEnabled(false);
        mWebView.setVerticalScrollBarEnabled(false);
    }

    @Override
    protected int attachLayout() {
        return R.layout.activity_about;
    }
}

File: app/src/main/java/cc/lixiaoyu/wanandroid/core/about/AboutActivity.kt
package cc.lixiaoyu.wanandroid.core.about

import android.os.Bundle
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import cc.lixiaoyu.wanandroid.databinding.ActivityAboutBinding
import cc.lixiaoyu.wanandroid.databinding.ActivityAboutBinding.*

class AboutActivity : AppCompatActivity() {

    private lateinit var binding: ActivityAboutBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = inflate(layoutInflater)
        setContentView(binding.root)
        initView()
    }

    private fun initView() {
        setSupportActionBar(binding.aboutToolbar)
        supportActionBar?.let {
            it.setDisplayHomeAsUpEnabled(true)
            it.setHomeButtonEnabled(true)
        }
        // 初始化 webview
        binding.aboutWebview.let {
            it.loadUrl("file:///android_asset/about_us.html")
            it.isHorizontalScrollBarEnabled = false
            it.isVerticalScrollBarEnabled = false
        }
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == android.R.id.home) {
            finish()
        }
        return true
    }
}